docs/changelog.md

```diff
@@ -213,6 +233,10 @@ See documentation, to learn more about using new API.
 
 ### Changed
 
+- Working directory for tests was changed to a directory where tests are located
+- Install `gtest` and `benchmark` via Conan or OS package managers instead of using them as submodules
+- Location of the sc-machine build tree, binaries, libraries and extensions
+- Up minimum required cmake version to 3.24.0
```

Comment:
```suggestion
- Working directory for each test has been changed to the test's source dir
- `gtest` and `benchmark` are installed via Conan or OS package managers instead of git submodules
- Location of the sc-machine build tree, binaries, libraries and extensions has been changed
- Bumped minimum required cmake version to 3.24.0
```